webapp/channels/src/i18n/en.json

```diff
@@ -65,15 +65,15 @@
   "activity_log.moreInfo": "More info",
   "activity_log.os": "OS: {os}",
   "activity_log.sessionId": "Session ID: {id}",
-  "activity_log.sessionsDescription": "Sessions are created when you log in through a new browser on a device. Sessions let you use Mattermost without having to log in again for a time period specified by the System Admin. To end the session sooner, use the 'Log Out' button.",
+  "activity_log.sessionsDescription": "Sessions are created when you log in through a new browser on a device. Sessions let you use Mattermost without having to log in again for a time period specified by the system administrator. To end the session sooner, use the 'Log Out' button.",
```

Comment:
For consistency across docs, training, and product, use "system admin" over "system administrator"

Reply:
`system admin` or `System Admin` (that is... what casing we prefer here?)

Reply:
We've steadily been moving towards lowercase: ``system admin``